lib/hanami/model/migrator/connection.rb

```diff
@@ -81,31 +85,39 @@ def password
         # @since 0.5.0
         # @api private
         def uri
-          adapter_connection.uri
+          raw.uri
         end
 
         # Returns DB connection wihout specifying database name
         #
         # @since 0.5.0
         # @api private
         def global_uri
-          adapter_connection.uri.sub(parsed_uri.select(:path).first, '')
+          raw.uri.sub(parsed_uri.select(:path).first, '')
         end
 
         # Returns a boolean telling if a DB connection is from JDBC or not
         #
         # @since 0.5.0
         # @api private
         def jdbc?
-          !adapter_connection.uri.scan('jdbc:').empty?
+          !raw.uri.scan('jdbc:').empty?
         end
 
         # Returns database connection URI instance without JDBC namespace
         #
         # @since 0.5.0
         # @api private
         def parsed_uri
-          @uri ||= URI.parse(adapter_connection.uri.sub('jdbc:', ''))
+          @uri ||= URI.parse(raw.uri.sub('jdbc:', ''))
+        end
+
+        # Return the database table for the given name
+        #
+        # @since x.x.x
+        # @api private
+        def table(name)
+          raw[name] if raw.tables.include?(name)
```

Comment:
@AMHOL Nope, this is a Sequel object which would raise an exception for unknown tables. 

We can't let this to fail because the caller is usually an object that backs CLI commands. For instance `hanami db version`. If there isn't migrations table we won't raise an exception, but just tell the user that we're still at version 0.